x-pack/test/apm_api_integration/tests/metrics/serverless_overview.spec.ts

```diff
@@ -0,0 +1,406 @@
+                };
+              })
+            ).to.eql(
+              pythonServerlessFunctionNames.map((name) => {
+                return { serverlessFunctionName: name, instance: PYTHON_INSTANCE_NAME };
+              })
+            );
+          });
+
+          it('returns correct values for all serverless functions', () => {
+            pythonServerlessFunctionNames.forEach((name) => {
+              const activeInstanceOverview =
+                serverlessMetrics.serverlessActiveInstancesOverview.find(
+                  (item) => item.serverlessFunctionName === name
+                );
+
+              expect(activeInstanceOverview?.serverlessDurationAvg).to.eql(FAAS_DURATION);
+              expect(activeInstanceOverview?.serverlessDurationAvg).to.eql(BILLED_DURATION_MS);
+              expect(activeInstanceOverview?.avgMemoryUsed).to.eql(expectedMemoryUsed);
+              expect(activeInstanceOverview?.memorySize).to.eql(MEMORY_TOTAL);
+            });
+          });
+
+          describe('timeseries', () => {
+            it('returns correct mean values', () => {
+              pythonServerlessFunctionNames.forEach((name) => {
+                const activeInstanceOverview =
+                  serverlessMetrics.serverlessActiveInstancesOverview.find(
+                    (item) => item.serverlessFunctionName === name
+                  );
+
+                function removeNullAndZeroValue(coordinate: Coordinate) {
+                  return coordinate.y !== null && coordinate.y !== 0;
+                }
+
+                const meanDuration = meanBy(
+                  activeInstanceOverview?.timeseries.serverlessDuration.filter(
+                    removeNullAndZeroValue
+                  ),
+                  'y'
+                );
+                expect(meanDuration).to.eql(FAAS_DURATION);
+
+                const meanBilledDuration = meanBy(
+                  activeInstanceOverview?.timeseries.billedDuration.filter(removeNullAndZeroValue),
+                  'y'
+                );
+                expect(meanBilledDuration).to.eql(BILLED_DURATION_MS);
+
+                const meanAvgMemoryUsed = meanBy(
+                  activeInstanceOverview?.timeseries.avgMemoryUsed.filter(removeNullAndZeroValue),
+                  'y'
+                );
+                expect(meanAvgMemoryUsed).to.eql(expectedMemoryUsed);
+
+                const meanMemorySize = meanBy(
+                  activeInstanceOverview?.timeseries.memorySize.filter(removeNullAndZeroValue),
+                  'y'
+                );
+                expect(meanMemorySize).to.eql(MEMORY_TOTAL);
+              });
+            });
+          });
+        });
+
+        describe('metrics', () => {
+          it('returns all metrics chart', () => {
+            expect(serverlessMetrics.metricCharts.length).to.be.greaterThan(0);
+            expect(serverlessMetrics.metricCharts.map(({ title }) => title).sort()).to.eql([
```

Comment:
Could also use `.contains` without sorting

Reply:
do you mean `expect().contain()`?

Reply:
Yes, so it doesn't need sorting - but it's up to you

Reply:
I got 

```
Error: expected [ 'Avg. Duration',
   'System memory usage',
   'Cold start duration',
   'Cold start',
   'Compute usage' ] to contain [ 'Avg. Duration',
   'System memory usage',
   'Cold start duration',
   'Cold start',
   'Compute usage' ]
```

when I changed to use contain instead:
```
expect(serverlessMetrics.charts.map(({ title }) => title)).to.contain([
          'Avg. Duration',
          'System memory usage',
          'Cold start duration',
          'Cold start',
          'Compute usage',
        ]);
```